sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md

```diff
@@ -519,7 +519,7 @@ Please follow [instructions here](https://github.com/Azure/azure-sdk-for-java/bl
 [refdocs]: https://azure.github.io/azure-sdk-for-java/springboot.html#azure-spring-boot
 [package]: https://mvnrepository.com/artifact/com.azure.spring/azure-spring-boot-starter-active-directory-b2c
 [sample]: https://github.com/Azure-Samples/azure-spring-boot-samples
-[logging]: https://github.com/Azure/azure-sdk-for-java/wiki/Logging-with-Azure-SDK#use-logback-logging-framework-in-a-spring-boot-application
+[logging]: https://docs.microsoft.com/en-us/azure/developer/java/sdk/logging-overview
```

Comment:
This will fail CI, and will need to be changed to https://docs.microsoft.com/azure/developer/java/sdk/logging-overview (without the en-us)